content/blog/2018/aws-nlb/index.md

```diff
@@ -81,32 +82,19 @@ You need to apply policy on the master role in order to be able to provision net
 
 1. Your policy is now attach to your master node.
 
-## Rewrite Istio Ingress Service
+## Generate the Istio manifest
 
-You need to rewrite ingress service with the following:
```

Comment:
Probably would be good to add a sentence ahead of this one that explains that you need to annotate the istio-ingress service to use the NLB load balancer.